tests/testthat/test-5transformCounts.R

```diff
@@ -70,6 +70,112 @@ test_that("transformAssay", {
                          log2(x+5)
                      }), check.attributes = FALSE)
         
+        ############################ CSS ######################################
+        skip_if_not(require("metagenomeSeq", quietly = TRUE))
+        # Define counts matrix for the css and css.fast testing
+        counts_matrix <- as.matrix(assay(tse, "counts"))
```

Comment:
If this package is not available, it skips all tests below. These tests should be at the end.